src/lib.rs

```diff
@@ -1,8 +1,9 @@
 #![warn(missing_docs)]
+#![deny(unused_results)]
```

Comment:
I will change this from deny to warn in a commit. I don't want to opt in to be broken when Rustc changes to emit this lint in more places. Deny is good in CI but not in the released version.

Reply:
got it, cool with me :)